pkg/pipelinerun/description/description.go

```diff
@@ -28,6 +42,10 @@ const templ = `{{decorate "bold" "Name"}}:	{{ .PipelineRun.Name }}
 {{- if ne $timeout "" }}
 {{decorate "bold" "Timeout"}}:	{{ .PipelineRun.Spec.Timeout.Duration.String }}
 {{- end }}
+{{decorate "bold" "Labels"}}:
```

Comment:
Instead of showing the `Labels` mark here even if no labels are present, maybe it would make sense here to not show `Labels` at all if `.PipelineRuns.Labels` length is 0. 

Example

Instead of

```
Namespace:   ns
Task Ref:    t1
Timeout:     1h0m0s
Labels:
```

It would be

```
Namespace:   ns
Task Ref:    t1
Timeout:     1h0m0s
```

Reply:
Modified Thank you